app/src/main/kotlin/at/bitfire/davdroid/ui/AppSettingsScreen.kt

```diff
@@ -609,16 +617,23 @@ private fun PushDistributorSelectionDialog(
                             )
                         },
                         trailingContent = {
-                            icon?.let {
+                            if (isSelf)
                                 Image(
-                                    bitmap = icon.toBitmap().asImageBitmap(),
+                                    painter = painterResource(R.drawable.product_logomark_cloud_messaging_full_color),
                                     contentDescription = null,
                                     modifier = Modifier.size(32.dp)
                                 )
-                            }
+                            else
+                                icon?.let {
+                                    Image(
+                                        bitmap = icon.toBitmap().asImageBitmap(),
+                                        contentDescription = null,
```

Comment:
I know we don't do it often, but I think here it's actually important to add a description. Just the same as the headlineContent should be fine I think.

Doc on contentDescription:
"text used by accessibility services to describe what this icon represents. This should always be provided unless this icon is used for decorative purposes, and does not represent a meaningful action that a user can take. ..."